Updates to welcome wagon page for 2023

canncrochet opened the pull request:
General updates for 2023, back in-person, and new location.

<!-- readthedocs-preview writethedocs-www start -->
----
:books: Documentation preview :books:: https://writethedocs-www--1909.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

canncrochet committed:
Merge pull request #4 from canncrochet/canncrochet-patch-4

Update WW guide post Portland 2022 conference

canncrochet committed:
Merge pull request #5 from canncrochet/canncrochet-patch-4

Fixed issue with heading

canncrochet committed:
Merge pull request #6 from writethedocs/main

Update wtd fork to wtd main

canncrochet committed:
Update for in-person 2023

General updates for back to in-person and new rooms/location.

diggerlilly commented on docs/conf/portland/2023/welcome-wagon.rst:
```diff
@@ -3,5 +3,195 @@
+Sunday at 2:00pm
+Monday morning at 8:15am
+
+Join us for an informal Introduction to Write the Docs in the Library/Astoria Room. We’ll review information about the conference specifically for first-timers and give everyone a chance to meet someone new. Come on a short tour of the venue so you’ll know where everything is and everything you can take part in.
```
Maybe put this before the list of times

plaindocs replied:
I reckon not :-p (just to be awkward)

diggerlilly commented on docs/conf/portland/2023/welcome-wagon.rst:
```diff
@@ -3,5 +3,195 @@
+~~~~~~~~~~~~~~~~
+
+-  Snacks are provided for you on conference days.
+-  Coffee, tea, and water are always available in the Crystal Ballroom. Bring a water bottle to make it easier for you to stay hydrated.
```
s/Crystal Ballroom/(how are we describing the Show Bar refreshment area?)

canncrochet replied:
Thank you! According to the notes, it sounds like snacks will be in Library Room. Do you mean the fact that you could buy your own refreshments from Show Bar? I'm not sure when that is open. I thought it was only open during shows.

diggerlilly commented on docs/conf/portland/2023/welcome-wagon.rst:
```diff
@@ -3,5 +3,195 @@
+-  The Revolution Hall main stage will have tables and rows of seats behind them.
+-  There's seating in the balcony that you can access from the third floor.
+-  There are no reserved seats; feel free to sit anywhere.
+-  If you can, show up early to the conference each morning to grab a seat at one of the round tables. Introducing yourself to your neighbors is one of the easiest way to meet people.
```
just making sure this is still relevant for Rev Hall tables...

ericholscher replied:
Yea, we're working with them to get a few tables, but it won't be as many as in the Crystal for sure.

plaindocs commented on docs/conf/portland/2023/welcome-wagon.rst:
```diff
@@ -3,5 +3,195 @@
+
+Sunday at noon
+Sunday at 2:00pm
+Monday morning at 8:15am
```
```suggestion
- Sunday at noon
- Sunday at 2:00pm
- Monday morning at 8:15am
```

plaindocs commented on docs/conf/portland/2023/welcome-wagon.rst:
```diff
@@ -3,5 +3,195 @@
+-  In the morning, or when you need a break during the day, head down to the Library/Astoria Room to check out the unconference schedule. Make note of any unconference talks you want to attend.
+-  Check out the lightning talks, and get excited about presenting one at next year's conference.
+
+Sample strategy for a second or higher Write the Docs conference
```
```suggestion
Sample strategy for a second or subsequent Write the Docs conference
```

plaindocs reviewed (approved):
Looks great overall

mopdx commented:
Looks great!

canncrochet committed:
Update docs/conf/portland/2023/welcome-wagon.rst

Co-authored-by: Sam <lykoszine@gmail.com>

canncrochet committed:
Update docs/conf/portland/2023/welcome-wagon.rst

Co-authored-by: Sam <lykoszine@gmail.com>

canncrochet committed:
Incorporating review feedback

canncrochet committed:
Merge branch 'main' into canncrochet-patch-5

canncrochet committed:
Fix title underline error

canncrochet committed:
Incorporate feedback from PR#1909 review and add WW Help desk

Incorporate feedback from PR#1909 review that I had to close and add WW Help desk

canncrochet committed:
Fix some formatting issues and replace crew images with links to twitter

canncrochet commented:
I messed up this PR, so closing. Incorporating all feedback here into a new PR I'll make shortly.
